### ISSUE TITLE
Dbt does not works due to bug in psycopg2-2.7.1

### ISSUE THREAD

**Issue by sphinks** [fishtown-analytics/dbt#773]
There is an issue in psycopg2, which I faced with after clean install of dbt on Fedora 26:
https://github.com/psycopg/psycopg2-wheels/issues/2
After installation I try to run 'dbt init <project_name>' and get an error:

```Traceback (most recent call last):
  File "/bin/dbt", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/ivan/Projects/dbt/scripts/dbt", line 3, in <module>
    import dbt.main
  File "/home/ivan/Projects/dbt/dbt/main.py", line 10, in <module>
    import dbt.project as project
  File "/home/ivan/Projects/dbt/dbt/project.py", line 13, in <module>
    import dbt.context.common
  File "/home/ivan/Projects/dbt/dbt/context/common.py", line 6, in <module>
    from dbt.adapters.factory import get_adapter
  File "/home/ivan/Projects/dbt/dbt/adapters/factory.py", line 3, in <module>
    from dbt.adapters.postgres import PostgresAdapter
  File "/home/ivan/Projects/dbt/dbt/adapters/postgres/__init__.py", line 1, in <module>
    from dbt.adapters.postgres.impl import PostgresAdapter
  File "/home/ivan/Projects/dbt/dbt/adapters/postgres/impl.py", line 1, in <module>
    import psycopg2
  File "/usr/lib64/python2.7/site-packages/psycopg2/__init__.py", line 50, in <module>
    from psycopg2._psycopg import (                     # noqa
ImportError: /usr/lib64/python2.7/site-packages/psycopg2/.libs/libresolv-2-c4c53def.5.so: symbol __res_maybe_init, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```
I get sources and switch in setup.py version of psycopg2 to 2.7.4 (where bug already fixed), install dbt from sources and dbt seems to be working great now.

Not sure if helps only me or it fix the root cause of problem, so I create an issue, instead of pull request with new version of psycopg2.

**Comment by drewbanin** [comment]
Thanks for the report @sphinks! I haven't seen this happen to anyone else, and our psycopg2 version hasn't changed for a long time, so I don't have a great handle on why this would happen.

It would be good to update to the latest release of pyscopg2. We do have an issue to use the pyscopg2 wheel explicitly (https://github.com/fishtown-analytics/dbt/issues/704), and we can bump the version at the same time.

**Comment by michael-erasmus** [comment]
I'm getting this error too, I did try running 

`pip install --no-binary :all: psycopg2`

As suggested [here](https://github.com/psycopg/psycopg2-wheels/issues/2#issuecomment-324599740) but I'm still seeing the error.

I'm running on Arch Linux.

**Comment by michael-erasmus** [comment]
Small update, I think this was indeed related to the version of psycopg2. I managed to work around this issue by bumping the version up to the latest. More details over here: https://github.com/fishtown-analytics/dbt/issues/704

**Comment by drewbanin** [comment]
Thanks for the update @michael-erasmus. I'm pretty surprised that both you and @sphinks were having issues, but maybe that's due to your Fedora and Arch OS's?

I'm very open to updating the psycopg2 dep -- let's discuss in the issue you linked (#704). Let's leave this one open until we get the changes merged

**Comment by sphinks** [comment]
Hm, does arch linux and fedora have so much similar, to get the same issue to due OS only?

**Comment by drewbanin** [comment]
@sphinks not sure! I don't have either of those OS's handy, so it's difficult for me to say for certain. I suspect it has more to do with the versions of libc that you two are using, or possibly some OS-specific code in psycopg2. I don't think that this problem occurs on macOS or Windows.

Regardless, I just made a PR to bump the dep to 2.7.5 which should hopefully fix this issue. If you have the time, I'd love to hear if installing from the `bump/psycopg2-2.7.5` branch works for you! See: https://github.com/fishtown-analytics/dbt/pull/807

**Comment by butla** [comment]
@drewbanin I have the same issue on Ubuntu 17.10, so it seems that this is a pretty wide problem. It's fixed when I install psycopg2==2.7.5.

**Comment by drewbanin** [comment]
Thanks @butla - this was fixed in #807 and is going out in the next release of dbt, 0.10.2!
